physics/sfcsub.F

```diff
@@ -1128,6 +1159,10 @@ subroutine sfccycle(lugb,len,lsoil,sig1t,deltsfc                    &
 !
 !  scale surface roughness and albedo to model required units
 !
+!     do i=1,len
```

Comment:
If you want to keep print statements, please put them inside an `if` block that only gets executed when a debug flag is set or something in order to keep default model output cleaner.